### PR TITLE
Update README.md and AUTHORS.txt for 14.2.0

### PR DESCRIPTION
**Pull request by yantosca**
### Name and Institution (Required)

Name: Bob Yantosca
Institution: Harvard + GCST

### Confirm you have reviewed the following documentation

- [x] [Contributing guidelines](https://geos-chem.readthedocs.io/en/stable/reference/CONTRIBUTING.html)

### Describe the update

1. `README.md`: Replaced links to defunct site `acmg.seas.harvard.edu/geos` with links to `geos-chem.org`.
2. `AUTHORS.txt`: Added new authors for 14.2.0.  Also now display names as "LAST, First", which makes searching easier.

### Expected changes

This will be a zero-diff update w/r/t model results.  It will only affect how `AUTHORS.txt` and `README.md` render.

### CODE REVIEW

File: AUTHORS.txt [hunk 1]
```diff
@@ -1,354 +1,363 @@
...
+KOUATCHOU, Jules
+KREMSER, Stefanie
+LACEY, Forrest
+LAMENCUSA, Carmen
+LAMSAL, Lok
+LAPINA, Kateryna
+LARSON, Kelsey
+LARY, David
+LATIMER, Robyn
+LE, Philippe Sager
```
Comment (msulprizio) [inline AUTHORS.txt:152]: Change to "LE SAGER, Philippe"

File: AUTHORS.txt [hunk 2]
```diff
@@ -1,354 +1,363 @@
...
+JANSEN, Ruud
+JIANG, Weiyuan
+JIANG, Zhe
+JIN, Lixu
+JISKRA, Martin
+JOHNSON, Matthew
+JONES, Dylan
+JUNG, Jaegun
+KASIBHATLA, Prasad
+HUDMAN, Rynda Kay
```
Comment (msulprizio) [inline AUTHORS.txt:133]: Change to "HUDMAN KAY, Rynda"

File: AUTHORS.txt [hunk 3]
```diff
@@ -1,354 +1,363 @@
...
+DOMMERGUE, Aurelien
+DOWNS, Will
+DRURY, Easan
+DUNCAN, Bryan
+EASTHAM, Sebastian
+ELLIS, Raluca
+ENBERG, Joseph
+ESTRADA, Lucas
+EVANS, Mathew
+FAIRLIE,
```
Comment (msulprizio) [inline AUTHORS.txt:77]: Change to "FAIRLIE, T. Duncan"

File: AUTHORS.txt [hunk 4]
```diff
@@ -1,354 +1,363 @@
...
+CRUZ, Xochitl
+CURCI, Gabriele
+DACHS, Jordi
+DAVILA, Yanko
+DELWICHE, Kyle
+DENG, Feng
+DESOUZA, Priyanka
+DESSERVETTAZ, Max
+DEUTSCHER, Nicholas
+DI, Maurizio Pierro
```
Comment (msulprizio) [inline AUTHORS.txt:66]: Change to "DI PIERRO, Maurizio"

File: AUTHORS.txt [hunk 5]
```diff
@@ -1,354 +1,363 @@
...
+BOYS, Brian
+BRATTICH, Erika
+BREIDER, Tom
+BREWER, Jared
+BROWNE, Ellie
+BROWN, Benjamin-Steiner
+BUKOSA, Beata
+BUTENHOFF, Christopher
+CADY, Karen-Pereira
+CAMERON, Philip-Smith
```
Comment (msulprizio) [inline AUTHORS.txt:33]: Change to "CAMERON-SMITH, Philip"

File: AUTHORS.txt [hunk 6]
```diff
@@ -1,354 +1,363 @@
...
+BOWMAN, Kevin
+BOYS, Brian
+BRATTICH, Erika
+BREIDER, Tom
+BREWER, Jared
+BROWNE, Ellie
+BROWN, Benjamin-Steiner
+BUKOSA, Beata
+BUTENHOFF, Christopher
+CADY, Karen-Pereira
```
Comment (msulprizio) [inline AUTHORS.txt:32]: Change to "CADY-PEREIRA, Karen"

File: AUTHORS.txt [hunk 7]
```diff
@@ -1,354 +1,363 @@
...
+BOJJAGANI, Sreekanth
+BONILLA, Eimy
+BOVY, Benoit
+BOWMAN, Kevin
+BOYS, Brian
+BRATTICH, Erika
+BREIDER, Tom
+BREWER, Jared
+BROWNE, Ellie
+BROWN, Benjamin-Steiner
```
Comment (msulprizio) [inline AUTHORS.txt:29]: Change to "BROWN-STEINER, Benjamin"

**Review by msulprizio — CHANGES_REQUESTED** [review]
There are several typos in `AUTHORS.txt` that need to be fixed.

File: AUTHORS.txt [hunk 8]
```diff
@@ -1,354 +1,363 @@
...
+FISHER, Jenny
+FITE, Charley
+FLEMING, Eric
+FLO, Judit-Gaya
+FORD, Bonne
+FRANCIS, Timmy
+FREESE, Lyssa
+FRIEDMAN, Carey
+FRITZ, Thibaud
+FU,
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Now fixed in commit 272b8d2

File: AUTHORS.txt [hunk 9]
```diff
@@ -1,354 +1,363 @@
...
+FARINA, Sal
+FAYE, Vivian
+FEINBERG, Ari
+FIELD, Brendan
+FIORE, Arlene
+FISCHER, Emily
+FISHER, Jenny
+FITE, Charley
+FLEMING, Eric
+FLO, Judit-Gaya
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Now fixed in commit 272b8d2

File: AUTHORS.txt [hunk 10]
```diff
@@ -1,354 +1,363 @@
...
+LARY, David
+LATIMER, Robyn
+LE, Philippe Sager
+LEE, Chulkyu
+LEE, Colin
+LEE, Meemong
+LEHMANN, Ralph
+LEIBENSPERGER, Eric
+LERICHE, Maud
+LEUNG,
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Now fixed in commit 272b8d2

File: AUTHORS.txt [hunk 11]
```diff
@@ -1,354 +1,363 @@
...
+LI, Bengang
+LI, Chi
+LI, Ke
+LI, Qinbin
+LI, Xianglan
+LI, Yanshun
+LIAO,
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Now fixed in commit 272b8d2

File: AUTHORS.txt [hunk 12]
```diff
@@ -1,354 +1,363 @@
...
+LIANG, Qing
+LIAO, Hong
+LIN, Haipeng
+LIN, Jintai
+LIN,
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Now fixed in commit 272b8d2

File: AUTHORS.txt (hunk 3 above)
Comment (yantosca) [reply AUTHORS.txt]: Now fixed in commit 272b8d2

File: AUTHORS.txt (hunk 6 above)
Comment (yantosca) [reply AUTHORS.txt]: Now fixed in commit 272b8d2

File: AUTHORS.txt [hunk 13]
```diff
@@ -1,354 +1,363 @@
...
+MARVIN, Maggie
+MCDUFFIE, Erin
+MCGUFFIN, Dana
+MCLINDEN, Chris
+MENG, Hongjian
+MENG, Jun
+MESKHIDZE, Nicholas
+MIATSELSKAYA, Natallia
+MICKLEY, Loretta
+CHAN, Christopher Miller
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Fixed in 8c1bfd3

File: AUTHORS.txt (hunk 1 above)
Comment (yantosca) [reply AUTHORS.txt]: Fixed in 8c1bfd3

File: AUTHORS.txt (hunk 2 above)
Comment (yantosca) [reply AUTHORS.txt]: Fixed in 8c1bfd3

File: AUTHORS.txt [hunk 14]
```diff
@@ -1,354 +1,363 @@
...
+WEISENSTEIN, Debra
+WELLS, Kelley
+WENG, Hongjian
+WESTERVELT, Dan
+WHALEY, Cynthia
+WIEDINMYER, Christine
+WILD, Oliver
+WOHLTMANN, Ingo
+WU, Shiliang
+WU, Shiliang
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Fixed in 8c1bfd3

File: AUTHORS.txt [hunk 15]
```diff
@@ -1,354 +1,363 @@
...
+TRAVIS, Katherine
+TRAYANOV, Atanas 
+TRIVITAYANURAK, Win
+TRUJILLO, Corey
+TUCCELLA, Paolo
+TURNER, Alexander
+TURQUETY, Solene
+TZOMPA, Zitely-Sosa
+UNGER, Nadine
+VAL, Maria-Martin
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Fixed in 8c1bfd3

File: AUTHORS.txt [hunk 16]
```diff
@@ -1,354 +1,363 @@
...
+TIAN, Rong
+TOMBROU, Maria
+TRAVIS, Katherine
+TRAYANOV, Atanas 
+TRIVITAYANURAK, Win
+TRUJILLO, Corey
+TUCCELLA, Paolo
+TURNER, Alexander
+TURQUETY, Solene
+TZOMPA, Zitely-Sosa
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Fixed in 8c1bfd3

File: AUTHORS.txt [hunk 17]
```diff
@@ -1,354 +1,363 @@
...
+SELIN, Noelle
+SEYMOUR, Michael
+SHAH, Viral
+SHAO, Jingyuan
+SHEN, Lu
+SHERWEN, Tomas
+SHUTTER, Joshua
+SILVA, Sam
+SMITH, Nicole-Downey
+LAERKE, Anne Soerensen
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Fixed in 8c1bfd3

File: AUTHORS.txt [hunk 18]
```diff
@@ -1,354 +1,363 @@
...
+SEBASTIANELLI, Paolo
+SELIN, Noelle
+SEYMOUR, Michael
+SHAH, Viral
+SHAO, Jingyuan
+SHEN, Lu
+SHERWEN, Tomas
+SHUTTER, Joshua
+SILVA, Sam
+SMITH, Nicole-Downey
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Fixed in 8c1bfd3

File: AUTHORS.txt [hunk 19]
```diff
@@ -1,354 +1,363 @@
...
+PAGE, Neil
+PAI, Sidhant
+PALMER, Paul
+PARAZOO, Nick
+PARELLA, Justin
+PARK, Rokjin
+PARRINGTON, Mark
+PAULOT, Fabien
+PHILIP, Sajeev
+PICKETT, Christopher-Heaps
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Fixed in 8c1bfd3

File: AUTHORS.txt [hunk 20]
```diff
@@ -1,354 +1,363 @@
...
+MURPHY, Killian
+MURRAY, Lee
+NAM, Jun
+NASSAR, Ray
+NATHAN, Brian
+NENES, Athanasios
+NEVISON, Cynthia
+NIELSEN, Eric
+NOTHOLT, Justus
+O, Kate'Dell
```
[the remark this answers is not in the document]
Comment (yantosca) [reply AUTHORS.txt]: Fixed in b8a1af7

**Review by msulprizio — APPROVED** [review]
Thanks @yantosca! These updates look good to merge now.

**Pull request merged**